### ISSUE TITLE
documents app can not be enabled

### ISSUE THREAD

**Issue by dttpy** [owncloud/documents#445]
### Steps to reproduce
1. go to apps page
2. click "Recommended" on the sidebar
3. find app "Documents 8.0" and click enable 

### Expected behaviour
Documents app can be enabled

### Actual behaviour
App can't be installed because the version in info.xml/version is not the same as the version reported from the app store

### Server configuration
**Operating system:** Ubuntu 13.10

**Web server:** apache 2.4.6 (Ubuntu)

**Database:** SQLite

**PHP version:** 5.5.3-1ubuntu2.3

**ownCloud version:** 8.0.0-beta2

**Updated from an older ownCloud or fresh install:** owncloud 7.0.4

**List of activated apps:** standard apps of owncloud

**The content of config/config.php:**

```
<?php
define('DEBUG',true);
$CONFIG = array (
  'instanceid' => 'oc6d9cda6e26',

  'datadirectory' => '/var/www/owncloud8-beta2/data',
  'dbtype' => 'sqlite3',
  'version' => '8.0.0.4',
  'installed' => true,
  'theme' => '',
  'maintenance' => false,
  'maxZipInputSize' => 838860800,
  'allowZipDownload' => true,
  'loglevel' => '0',
  'trusted_domains' => 
  array (
    0 => '192.168.1.80',
  ),
  'appcodechecker' => false,

  );



```

**Are you using external storage, if yes which one:** no
**Are you using encryption:** yes

### Client configuration
**Browser:** firefox 35.0.1

**Operating system:** ubuntu 13.10

**Comment by RealRancor** [comment]
Hi,

do you still had the documents app installed in /apps from OC 7.0.4?

**Comment by dttpy** [comment]
yes, I had activated documenets app in OC 7.0.4, but after update, I've noticed that some apps are no more there, so I've tried to install them again, but unfortunately, it doesn't work

**Comment by RealRancor** [comment]
Hi,

have followed the upgrade instructions in the docs and deleted all files of your installation except data and config?

**Comment by dttpy** [comment]
yes, I think so

**Comment by PVince81** [comment]
I have the same issue. I upgraded the OC 7.0.4 openSUSE RPM to 8.0.0~beta2 and the documents, calendar and contacts app disappeared, which I believe is expected.

Then I tried to enabled all three from the app store. Both calendar and contacts can be enabled (code checked disabled), but the documents app failed:
`App can't be installed because the version in info.xml/version is not the same as the version reported from the app store`

Maybe info.xml hasn't been adapted properly ?

@karlitschek @VicDeo
